presto-docs/src/main/sphinx/release/release-0.203.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.203
+=============
+
+General Changes
+---------------
+
+* Fix spurious duplicate key errors from :func:`map`.
+* Fix planning failure when correlated subqueries containing a ``LIMIT``
```

Comment:
I thnink singular will be easier to read "Fix planning failure when a correlated subquery containing a ``LIMIT`` clause is used within ``EXISTS`` (:issue:`10696`).